src/core/vectorlayercounterpart.js

```diff
@@ -1,4 +1,4 @@
-goog.provide('olcs.core.OlLayerPrimitive');
+goog.provide('olcs.core.VectorLayerCounterpart');
```

Comment:
We are not heriting from `Cesium.Primitive` anymore so it would be misleading to keep the `Primitive` suffix. Moreover, this type is specific to Vector layers: skipping the keyword would be ambiguous.